### PR TITLE
I had made the ANN(artificial neural network) model for #11

### PR DESCRIPTION
**Pull request by Kota-Karthik**
## Related Issue
#11 
- Info about the related issue
- This was a issue in which i had to create a model that classifies the fruits into respective categories
- I had made the model using ANN
-

- [x] GSSOC Participant
- [x] Contributor

Closes: #issue number that will be closed through this PR
#11 
### Describe the changes you've made

[When using Artificial Neural Networks (ANN) for fruit image classification, one modification that can be made is to use a cascaded forward network to grade and classify fruit images based on obtained feature values such as color, shape, and size](https://www.researchgate.net/publication/286933975_Quality_Detection_of_Fruits_by_Using_ANN_Technique)[1](https://www.researchgate.net/publication/286933975_Quality_Detection_of_Fruits_by_Using_ANN_Technique). [The accuracy of the fruit quality detection can be increased by combining these features with an ANN model](https://www.researchgate.net/publication/286933975_Quality_Detection_of_Fruits_by_Using_ANN_Technique)[1](https://www.researchgate.net/publication/286933975_Quality_Detection_of_Fruits_by_Using_ANN_Technique). This is just one example of how ANN can be used for fruit image classification.

## Type of change

I have made the ANN model from the zero
I had first taken care of Nan values 
then feature scaled data
then spilited data into train and test sets 
In the end i made the model 
then trained it on training set 
then evaluated on testing set

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?
I used the model.evaluate() function to check the accuracy of model 
on an average the accuracy was approx 91%

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots

|      Original       |      Updated       |
| :-----------------: | :----------------: |
| Original Screenshot | Updated Screenshot |

### CODE REVIEW

**Comment by lcs2022026** [comment]
@adithya-s-k 
I completed my issue before my deadline 
checked it multiple times to ensure it has no errors
I even calculated the avg accuracy of model ,which came out to be approximately 91%
Please kindly check the changes i had made and kindly merge into the main branch : )
